spring-cloud-gcp-data-datastore/src/main/java/org/springframework/cloud/gcp/data/datastore/core/DatastoreOperations.java

```diff
@@ -45,6 +45,13 @@
 	 */
 	<T> T save(T instance);
 
+	/**
+	 * Saves multiple instances of objects to Cloud Datastore. Behaves as update or insert.
+	 * @param entities the objects to save.
+	 * @return the entities that were saved.
+	 */
+	<T> Iterable<T> saveAll(Iterable<T> entities);
```

Comment:
I added this and also deleteAll to implement the CRUD repo deleteAll and saveAll functions. I wanted to take advantage of the client lib's version that can handle multiple entities, so I created these new functions.